core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-create/dot-experiments-create.component.ts

```diff
@@ -50,45 +46,30 @@ interface CreateForm {
     ],
     templateUrl: './dot-experiments-create.component.html',
     styleUrls: ['./dot-experiments-create.component.scss'],
-    providers: [DotExperimentsCreateStore],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DotExperimentsCreateComponent implements OnInit {
-    vm$: Observable<DotExperimentCreateStore> = this.dotExperimentsCreateStore.state$;
+    vm$: Observable<VmCreateExperiments> = this.dotExperimentsListStore.createVm$;
 
     form: FormGroup<CreateForm>;
 
-    /**
-     * Emited when the sidebar is closed
-     */
-    @Output()
-    closedSidebar = new EventEmitter<void>();
-    private pageId: string;
+    @Input()
```

Comment:
remove because the pageId comes in the `this.dotExperimentsListStore.createVm$` ?

Reply:
deleted